projects/legacy/components/color-selector/color-selector.template.html

```diff
@@ -69,7 +69,9 @@
             [iconStart]="getIcon(button)"
             [tuiHint]="button"
             (click)="onDirectionChange(button)"
-        ></button>
+        >
+            Direction
```

Comment:
We also need to add a rule that if `tuiHint` + `tuiHintDescribe` are there – we do not need text

Reply:
fixed